src/components/Note/Note.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import NoteBody from './styles/NoteBody';
+import NoteCorner from './styles/NoteCorner';
+
+const Note = ({ children, Body, Corner }) => (
+  <Body>
+    {children}
+    <Corner />
+  </Body>
+);
+
+Note.propTypes = {
+  /**
+   * Contents of the note
+   */
+  children: PropTypes.node.isRequired,
+  /**
+   * Override the component that renders the body container
+   */
+  Body: PropTypes.func,
```

Comment:
`Body` seems like it could be confused with HTML `body`. I don't have a better suggested name but it does seem like it could be confused

Reply:
Good point. I could go back to the typical 'Container' naming, but that seems unintuitive here.